personal/privacyidea/default/mfa_intro.tpl

```diff
@@ -237,7 +244,16 @@
                         <td><button class="txtonlybtn" type="submit" name="mfaTokenAction[mfaTokenView]" value="{$token.serial}">
                             {if strpos($tokenDescriptionACL, "r") !== false}{$token.description}{else}{t}not shown{/t}{/if}
                         </button></td>
-                        <td>{if strpos($tokenLastUsedACL, "r") !== false}{$token.info.last_auth}{else}{t}not shown{/t}{/if}</td>
+
+                        {* token.info.last_auth *}
+                        {if !empty($token.info.last_auth)}
+                            <td class="tokenLastUsed">
+                                {if strpos($tokenLastUsedACL, "r") !== false}<input type="datetime-local" readOnly value="{$token.info.last_auth}">{else}{t}not shown{/t}{/if}
```

Comment:
`readonly` is included in a submit, better use `disabled`.

Reply:
What exactly do you mean `readonly` is included in a submit?

`disabled` makes the input look gray.

Reply:
Both `readonly` does not prevent interaction, i.e. the value cannot be changed but the element still receives focus which we do not want here. Furthermore the value is still submitted. See https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly#attribute_interactions for details.

Reply:
After some testing with different browsers it has become clear the `<input type=datetime-local>` is unsuitable here as it cannot be styled. Firefox e.g. shows a reset icon even in disabled state and takes too much whitespace.